commerce/domain/promotion/cart/calculator/classes/PromotionCalculatorSample.cls

```diff
@@ -1,24 +1,53 @@
- /**
+/**
   * @description This sample is for the situations where Promotion Calculation needs to be extended or overridden via the
   * extension point for the Promotion Calculator. You are expected to refer this and write your own implementation.
   * This class must extend the CartExtension.PromotionsCartCalculator class to be processed.
+  *
+  * In this example cart items are evaluated against a BOGO (Buy X, Get Y) promotion > Buy 5 items of qualifying product X,
```

Comment:
Are there plans to add any CartExtension.CartAdjustmentGroup examples?  This example is a line level, but it would be nice to have an order level.  

Example 1 - Spend $100 and get 20% off
In this example I would imagine you get a CartExtension.CartAdjustmentGroup as well as CartExtension.CartItemPriceAdjustment with cartextension.CartAdjustmentTargetTypeEnum.ORDER.

Do you think this current example you have could also work with "Spend $75 and get free shipping" against the shipping cart item?

Reply:
@cthomas-dwre Based on discussion with our PM (Mofeyi Oluwalana), we decided to have product promotion example for now. And shipping promotion is not implemented in Apex in 250.